h2o-jetty-9/src/main/java/ai/h2o/org/eclipse/jetty/plus/jaas/spi/LdapLoginModule.java

```diff
@@ -0,0 +1,9 @@
+package ai.h2o.org.eclipse.jetty.plus.jaas.spi;
```

Comment:
We also need to support the `org.eclipse.jetty.plus.jaas.spi` in Jetty 9 to remain fully compatible